src/figuresClasses.ts

```diff
@@ -1,11 +1,90 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  constructor(
+    public color: string,
+    private a: number,
+    private b: number,
+    private c: number,
```

Comment:
The constructor parameters `a`, `b`, and `c` should be prefixed with `this.` when used inside the constructor to refer to the class properties. This will fix the reference error when checking the side lengths.